tiles/src/test/java/com/protomaps/basemap/layers/PlacesTest.java

```diff
@@ -99,4 +99,94 @@ void testMinMaxLabelPopulatedPlaceNoMatch() {
         0
       )));
   }
+
+  @Test
+  void testLocalityPopulationOsm() {
```

Comment:
None of these new tests pass, even though they contain the expected data as reported by the console. What gives?